src/kirby/components/slide-button/slide-button.common.scss

```diff
@@ -0,0 +1,7 @@
+$slide-button-container-width: get-unit-value(256);
```

Comment:
Align file name (everywhere else shared scss is called `_xxx.component.shared.scss`)